packages/flutter_hooks/lib/src/framework.dart

```diff
@@ -178,9 +178,9 @@ Calling them outside of build method leads to an unstable state and is therefore
     }
   }
 
-  /// Creates the mutable state for this hook linked to its widget creator.
+  /// Creates the mutable state for this [Hook] linked to its widget creator.
```

Comment:
According to https://dart.dev/guides/language/effective-dart/documentation#avoid-redundancy-with-the-surrounding-context, the class this method should be part of (`Hook` in this case) shouldn't be referenced here at all.

Reply:
I don't personally agree with this warning. I prefer using [class] even when redundant, because we get proper lints if somehow the class was renamed